CHIPTLV.h

```diff
@@ -0,0 +1 @@
+TLV.h
```

Comment:
Is it worth keeping these? Do we expect many external components include Matter's TLV headers directly?

Reply:
These feel odd to me as well ... are these expected to be here? They look like top level symlinks to me ... they feel odd.

Reply:
Yes, they are wrong.  I'll remove.